Extensions/EdFi.Ods.Profiles.Sample/EdFi.Ods.Profiles.Sample.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+<metadata>
+<id>EdFi.Ods.Profiles.Sample</id>
+<version>0.0.0</version>
+<authors>Ed-Fi Alliance</authors>
+<owners>Ed-Fi Alliance</owners>
+<projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Extensions</projectUrl>
+<requireLicenseAcceptance>false</requireLicenseAcceptance>
+<description>EdFi.Ods.Profiles.Sample</description>
+<releaseNotes></releaseNotes>
+<license type="file">LICENSE.txt</license>
+</metadata>
+<files>
+  <file src="../../../LICENSE.txt" />
```

Comment:
**QUESTION**: When I tried to build the nuget packages for these projects locally from the command-line, it couldn't find the LICENSE.txt file and I had to modify the path.

I'm wondering:
- Should `<file src="../../../LICENSE.txt" />` actually be `<file src="../../LICENSE.txt" />` (climb up 2 levels instead of 3)?
- Was I doing something "wrong" locally? (How is it that this path could be wrong on these projects and not result in build failures?)

Reply:
https://intedfitools1.msdf.org/buildConfiguration/OdsPlatform_OdsApi_Packages_EdFiOdsProfilesSample/113136?buildTab=artifacts
you can see license file in nuget file

Reply:
On the build server you're checking out the extensions into `Ed-Fi-Extensions/` but you license file is being checked out straight into the working directory for the build agent.
I think if you changed the checkout rules to checkout the license in the same `Ed-Fi-Extensions/` folder you could adjust the path here and it would work in both scenarios.

Reply:
Someone added a custom checkout rule which is putting the license file at the root on TeamCity, instead of in the Extensions repo.  This should probably be adjusted if we're using nuspec, we usually try to avoid doing TeamCity custom checkouts that will only work on the build server and won't work locally.
+:LICENSE.txt => LICENSE.txt